src/schema/v2/admin/mutations/createFeatureFlagMutation.ts

```diff
@@ -0,0 +1,167 @@
+import {
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLList,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLEnumType,
+  GraphQLNonNull,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { omit } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { FeatureFlags, FeatureFlagType } from "../featureFlags"
+
+export const FeatureFlagInputFields = {
```

Comment:
Pulled this out for now as we can share it for Update (if we decide to add that directly into Forque; not sure its necessary since there's links out to Unleash)